spec/backtrace_spec.rb

```diff
@@ -218,11 +216,35 @@
            function: 'realtime' }]
       end
 
-      it "returns a properly formatted array of hashes" do
-        stub_const('ExecJS::RuntimeError', AirbrakeTestError)
-        expect(
-          described_class.parse(ex, Logger.new('/dev/null'))
-        ).to eq(parsed_backtrace)
+      context "when not on Ruby 1.9" do
+        let(:ex) { ExecJS::RuntimeError.new.tap { |e| e.set_backtrace(bt) } }
+
+        it "returns a properly formatted array of hashes" do
+          stub_const('ExecJS::RuntimeError', AirbrakeTestError)
+          stub_const('Airbrake::RUBY_19', false)
+
+          expect(
+            described_class.parse(ex, Logger.new('/dev/null'))
+          ).to eq(parsed_backtrace)
+        end
+      end
```

Comment:
We do :) We didn't have a test for the failing scenario, so we never reached the failing check in our existing test.